claim/README.md

```diff
@@ -194,6 +194,20 @@ might be having with the ACLK or claiming process.
 
 Use these keys and the information below to troubleshoot the ACLK.
 
+#### bash: netdata-claim.sh: command not found
+
+If you run the claiming script and see a `command not found` error, the claiming script is not in your system's `$PATH`.
+Netdata expects this file at `/usr/sbin/`, but some systems may have installed the script in `/opt/netdata/usr/sbin/` or
```

Comment:
Netdata Cloud does, Netdata itself should be looking in the right place and if it isn’t then that’s a bug in netdata.

Reply:
Just read the long thread on Slack about this. Also related: https://github.com/netdata/netdata/issues/9202. I can't tell if these users are encountering a bug or not.

Perhaps the better route is to document that you may see this error if you installed Netdata in a non-standard location or are using an unsupported package. For the former, you can update your PATH or just find the script and run it from that location. For the latter, uninstall and reinstall using kickstart.

How does that sound?

Reply:
That sounds like an excellent approach.

Reply:
@Ferroin you have mentioned that you believe that we shouldn't makes changes to PATH and instead we should change the documents to point to absolute paths. Perhaps this is a good place to start?

Reply:
Possibly. The problem is that right now we don’t have absolute paths that work everywhere.

And it’s not an issue if _users_ decide to update the contents of `$PATH`, the issue that potentially arises is _us_ modifying it, because we can’t count on the user not changing it in some other way.